spec/models/pdc_metadata/resource_spec.rb

```diff
@@ -24,8 +22,6 @@
     expect(ds.main_title).to eq "hello world"
     expect(ds.resource_type).to eq "Dataset"
     expect(ds.creators.count).to be 2
-    expect(ds.creators.first.affiliations.count).to be 0
-    expect(ds.creators.second.affiliations.count).to be 1
```

Comment:
We currently do not support affiliations and therefore I removed these two tests. These tests tested functionality that we do not provide but they were here since serialization was free-standing before (i.e. not attached to the PDCSerialization::Datacite class). 

When/if we add affiliations to the workflow (e.g. allow a creator to be affiliated with an organization) then we can add test for it.